Getting-Started-with-Akka-http-signature.md

```diff
@@ -114,111 +114,77 @@ This will save the contents of the key in String format within the file found by
 
 ### Saving a public Key to RDF
 
-A user can also transform his public keys to an RDF. In order to do this one must first import multiple files in order to resolve the required dependencies.
+A user can also transform his public keys to an RDF. In order to do this one must first import multiple files in order to resolve the required dependencies:
```

Comment:
->"to an RDF Graph".

There is no such thing as _an RDF_.

Also I don't think we should be speaking of "user" here. Just stick to the generic _one_. Users are often thought of as end users, whereas here we are speaking of programmers, but that is obvious and a mouthful.